scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -150,7 +150,15 @@ private fun calculateAggregateCoverageReport(
     return coverages.find { it == Coverage.FULL } ?: Coverage.NONE
   }
 
-  val allCoveredLines = coverageReports.flatMap { it.coveredLineList }
+  val groupedCoverageReports = coverageReports.groupBy {
+    Pair(it.filePath, it.fileSha1Hash)
+  }
+
+  val singleCoverageReport = groupedCoverageReports.entries.single()
+  val (key, reports) = singleCoverageReport
```

Comment:
```suggestion
  val (key, reports) = groupedCoverageReports.entries.single()
```

Can be simplified a bit.

Reply:
Done, updated as suggested.